docs/sources/tempo/operations/monitor/set-up-monitoring.md

```diff
@@ -139,92 +138,97 @@ rule {
 
 This lets you create a configuration that scrapes metrics from Tempo components and writes the data to a Mimir instance of your choice.
 
-Here’s a complete configuration for a [Grafana Agent Helm](https://grafana.com/docs/agent/latest/flow/get-started/install/kubernetes/) values file for Flow, to scrape a running instance of Tempo.
+Here’s a complete configuration for a [Grafana Alloy Helm](https://grafana.com/docs/alloy/<ALLOY_VERSION>/configure/kubernetes/) values file for Alloy, to scrape a running instance of Tempo.
```

Comment:
This one converted to Alloy a bit oddly.

Reply:
I'm removing some of the 
```suggestion
This example provides a Helm values file for a [Grafana Alloy deployed on Kubernetes](https://grafana.com/docs/alloy/<ALLOY_VERSION>/configure/kubernetes/).
```